examples/platform_channel_swift/ios/.gitignore

```diff
@@ -0,0 +1,38 @@
+.idea/
+.vagrant/
+.sconsign.dblite
+.svn/
+
+.DS_Store
+*.swp
+*.lock
```

Comment:
is this intentionally different from https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/create/ios.tmpl/.gitignore ?